### PR TITLE
firebase integration; added docs for firebase setup; Basic get and se…

### PR DESCRIPTION
**Pull request by jbennyhuang**
…t with firebase;

Awaiting schema for further development

### CODE REVIEW

**Review by jx3yang — CHANGES_REQUESTED** [review]
I'd say remove the training data from source control, we all have access to it from google drive

File: backend/database_api/main.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	res, err := client.Collection(collection).Doc(document).Get(context.Background())
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	defer client.Close()
```
Comment (jx3yang) [inline backend/database_api/main.go]: Shouldn't `defer client.Close()` come before `res, err := client...`?

File: backend/database_api/main.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	}
+
+	res, err := client.Collection(collection).Doc(document).Set(context.Background(), quote)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	log.Print(res)
+
+	defer client.Close()
```
Comment (jx3yang) [inline backend/database_api/main.go]: Ditto

File: backend/database_api/main.go (hunk 1 above)
Comment (jbennyhuang) [reply backend/database_api/main.go]: done

File: backend/database_api/main.go (hunk 2 above)
Comment (jbennyhuang) [reply backend/database_api/main.go]: done

File: backend/database_api/go.mod [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module MAISHacks2020/backend/ping
```
Comment (jx3yang) [inline backend/database_api/go.mod]: Probably should be `database_api`

**Review by jx3yang — CHANGES_REQUESTED** [review]

File: backend/database_api/go.mod (hunk 3 above)
Comment (jbennyhuang) [reply backend/database_api/go.mod]: done

**Review by jx3yang — APPROVED** [review]
LGTM

**Comment by jbennyhuang** [comment]
all good ?

**Pull request merged**